static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -282,7 +282,7 @@ function BaseDraggableTabList({
         <MotionWrapper>
           {tempTab && (
             <Tab
-              key={tempTab.key}
+              key={'temporary-tab'}
```

Comment:
Probably best to reference a constant for this since multiple files are using the same string